src/forms/formHelpers.js

```diff
@@ -336,11 +342,12 @@ const RowMessage = function ({ validRow, message }) {
 
   var result = null;
   if (!validRow && message) {
-    result = <Label
+    result = 
+    <Label
```

Comment:
Same as above, I'd probably enclose the `Label` in parentheses.

Reply:
Parens here makes sense to me, too. I'm not conceding on the ternaries yet, though.